services/ui-src/src/components/fields/TextField.tsx

```diff
@@ -34,11 +34,12 @@ export const TextField = ({
     const fieldValue = form.getValues(name);
     if (fieldValue) {
       setDisplayValue(fieldValue);
+      form.setValue(name, fieldValue);
```

Comment:
Question: I don't think this would be necessary because on line 34 we get the field value from the form and then here we set the field value in the form to the value we just got from it. I'm guessing the fix still works without this?

Reply:
Still works. Initial fix of adding that did sometimes fix the issue but you are right in that really we just need the other line.